src/Sulu/Bundle/PageBundle/Resources/translations/admin.de.json

```diff
@@ -64,6 +64,7 @@
     "sulu_page.schedules_label": "Zeitfenster zuweisen",
     "sulu_page.schedules_description": "Der Block wird nur angezeigt, wenn mindestens einer der Bedingungen erfüllt wird.",
     "sulu_activity.resource.pages": "Seite",
+    "sulu_activity.resource.translation": "Übersetzung",
```

Comment:
think we should use `sulu_activity.resource.pages.translation` or something like `sulu_trash.restore_type.translation`/`sulu_trash.restore_type.pages.translation` here

Reply:
updated to `sulu_activity.resource.pages.translation`